test/built-ins/Temporal/PlainDateTime/prototype/toZonedDateTime/disambiguate-basic.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
This test seems to duplicate half of the multiple-instants.js test. I think it's probably OK to drop this one.

Reply:
Good point. Dropped.